packages/flutter_tools/lib/src/base/file_system.dart

```diff
@@ -17,6 +17,40 @@ import 'signals.dart';
 // the tool is killed by a signal.
 export 'package:file/file.dart';
 
+extension FileUtils on File {
```

Comment:
Not in love with the extension method, but I don't want to re-type everything as ErrorHandling file.

Maybe just a static helper method?

Reply:
Update to static method on ErrorHandlingFileSystem